src/snyk.ts

```diff
@@ -227,11 +227,11 @@ function projectIssuesToMarkdown(project: Project, projectIssues: ProjectIssues)
 
     for (const issue of projectIssues.issues) {
         console.log(issue)
-        markdownString += `\n\n#### [${issue.issueData.title}](${issue.issueData.url}) (Priority Score: ${issue.priority.score})\n- dependency: ${
-            issue.pkgName
-        }, version${issue.pkgVersions.length > 1 ? 's' : ''} ${issue.pkgVersions.join(', ')}\n- severity: ${
-            issue.issueData.severity
-        }\n\n`
+        markdownString += `\n\n#### [${issue.issueData.title}](${issue.issueData.url}) (Priority Score: ${
+            issue.priority.score
+        })\n- dependency: ${issue.pkgName}, version${issue.pkgVersions.length > 1 ? 's' : ''} ${issue.pkgVersions.join(
```

Comment:
I assume this is unrelated to the CORS issue – just noting my review is for the CORS update.

Reply:
This was automatically formatted on save, so I think we're good to publish